biblatex-iso690.tex

```diff
@@ -411,25 +543,21 @@ \subsubsection{Creators}
 individuals, the title should be used as the first element of the reference.
 However, it is appropriate to include some relevant roles, e.g. the director:
 
-%\begin{ltxexample}
-\begin{verbatim}
+\begin{ltxexample}
   ..
-  editora     = {Welles, Orson}
-  editoratype = {Directed by}
+  editora     = {Welles, Orson},
+  editoratype = {Directed by},
   ..
-\end{verbatim}
-%\end{ltxexample}
+\end{ltxexample}
 
 in English, or
 
-%\begin{ltxexample}
-\begin{verbatim}
+\begin{ltxexample}
   ..
-  editora     = {Welles, Orson}
-  editoratype = {Réžia}
+  editora     = {Welles, Orson},
+  editoratype = {Réžia},
```

Comment:
... here. With a key/bibstring approach there'd be no need to change to `Réžia` for Slovak.